x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/search/aggregations/bucket/geogrid/AbstractGeoHashGridTiler.java

```diff
@@ -23,6 +23,9 @@
     /** check if the provided hash is in the solution space of this tiler */
     protected abstract boolean validHash(String hash);
 
+    /** Max size of the solution space */
+    protected abstract long getMaxHashes();
```

Comment:
Would be nice to add a unit test for this one.

Reply:
done